src/main/scala/cromwell/util/EnhancedExponentialBackoff.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.util
+
+import com.google.api.client.util.ExponentialBackOff
+
+class EnhancedExponentialBackoff(enhancedBuilder: EnhancedExponentialBackoff.Builder) extends ExponentialBackOff(enhancedBuilder) {
+  private var firstCall: Boolean = true
```

Comment:
Does it?  I thought this was narrower but I haven't looked at it as closely as you have.  TryUtil has a known issue with not jittering (DSDEEPB-1915) that causes unnecessary pain when operations that fail for hitting a QPS limit are retried in sync.  So whatever we end up with should have at least as much jittering as we have now.  :smile: